core/src/main/ruby/jruby/truffle/core/float.rb

```diff
@@ -8,8 +8,18 @@
 
 class Float
 
-  NAN = 0.0 / 0.0
-  INFINITY = 1.0 / 0.0
-  EPSILON = 2.2204460492503131e-16
+  NAN        = 0.0 / 0.0
+  INFINITY   = 1.0 / 0.0
+  EPSILON    = 2.2204460492503131e-16
+  RADIX      = 2
+  ROUNDS     = 1
+  MIN        = 2.2250738585072014e-308
```

Comment:
This might break a spec since we are considered as JRuby in RubySpec and the JVM seems to always consider subnormals.